Purchases/Misc/PlatformInfo.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PlatformInfo.swift
+//
+//  Created by Josh Holtz on 2/17/22.
+
+import Foundation
+
+// swiftlint:disable missing_docs
+public extension Purchases {
+
+    @objc(RCPlatformInfo)
+    final class PlatformInfo: NSObject {
```

Comment:
I think this work, but it would be a bit clearer to put the `public` here instead of the extension.

Reply:
I switched back and forth three times on where it should go 🤦‍♂️